lib/kafo/scenario_manager.rb

```diff
@@ -29,17 +29,18 @@ def available_scenarios
     end
 
     def list_available_scenarios
-      say "Available scenarios"
+      say ::HighLine.color("Available scenarios", :cyan)
```

Comment:
please use [custom defined colors](https://github.com/theforeman/kafo/blob/master/lib/kafo/color_scheme.rb#L34) in the whole PR, the reason is that colors are changed according to color scheme (black vs white background)

feel free to add new key to color hash when needed...